.github/workflows/tests.yml

```diff
@@ -388,6 +388,11 @@ jobs:
 
       - name: Run detect regressions
         run: |
+          if [[ ${{ github.event_name }} = 'pull_request' ]]
+          then
+            export IS_PR='true'
+          fi
+          echo "IS_PR=$IS_PR"
```

Comment:
I think we need to put this into `$GITHUB_ENV` to have it persist across steps (see, e.g., the old "maybe mark for benchmarking")

Reply:
We would if we need it for in other steps but we don't use it anywhere else, that's why I didn't put it into $GITHUB_ENV. Unless I'm missing something here.

Reply:
You're absolutely right, my mistake